README.md

```diff
@@ -50,16 +70,9 @@
 |-----------------------|-------|
 | [Covapp](https://covapp.charite.de/) | Questionnaire in EN /DE, can be combined with app/database for self-reporting |
 | [Apple COVID-19 Watch](https://www.apple.com/newsroom/2020/03/apple-releases-new-covid-19-app-and-website-based-on-CDC-guidance/) | |
+| [WHO App](https://github.com/WorldHealthOrganization/) | Unreleased |
```

Comment:
I am surprised that the WHO App doesn't rate being bolded. Their github seems pretty active. I lurk on their slack and they have a lot of fairly active collaborators.